compose/runtime/runtime/build.gradle

```diff
@@ -116,9 +116,16 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
             commonMain.dependencies {
                 implementation(libs.kotlinStdlibCommon)
                 implementation(libs.kotlinCoroutinesCore)
+                implementation(project(":performance:performance-annotation"))
+
+                // Keep both project and module dependency for pseudo-relocated modules.
```

Comment:
It looks like we don't have to add dependencies on `androidx` artifacts until we remove the project dependency?

1. Every project now depends on our dumb artifact
2. Every project depends on `androidx` artifact via transitive dependency
3. When we decide to remove the dumb artifact (breaking binary compatibility for old versions), we just replace the project dependency by `androidx`.

P.S. If something in my logic is wrong, and we still have to do it, we have to do it everywhere, not only in `compose:runtime`

Reply:
Currently we don't need to add a module dependency on "androidx.collection/annotation:version".
It will be added transitively thanks to the dependecy on the stub project/module.
So I removed it for now.

___

But when we decide to stop publishing the stub, we'll need to add an explicit version dependency.